### ISSUE TITLE
Update FFMPEG for avoton.

### ISSUE THREAD

**Issue by dajoni** [SynoCommunity/spksrc#3021]
It seems the latest ffmpeg for my DS415+ (avoton) is 3.1.2-4.

I'd love to see it match the ffmpeg on my DS216+ (braswell) v. 3.3.3-7.

Is that possible?

**Comment by cytec** [comment]
@dajoni can you check if it shows up now?

**Comment by dajoni** [comment]
Doesn't seem like it.

I go to Package Center -> Installed -> ffmpeg and it still shows "Newest version" as 3.1.2-4

Is there any way to do a force refresh of the package center? The regular refresh button does not seem to cut it. Maybe related: On https://synocommunity.com/package/ffmpeg the avoton architecture is not mentioned.

**Comment by cytec** [comment]
uh sorry my fault. got confused and mixed up the arches...
seems like avoton wasn't published for some reason... also some cross packages seems to need an update since downloading them fails... I'm gonna look into that but that might take some days :/

**Comment by ymartin59** [comment]
ffmpeg has to be built with DSM 6.1 toolchain to run, but DSM 6 packages cannot be published on repository yet. Here is a package that should work on avoton: http://dl.free.fr/jkXq6ZrsV

**Comment by cytec** [comment]
uhm, @ymartin59 isn't there a DSM 5 toolchain for avoton already? thought so, as the 3.1.2 version was compiled for avoton just the most recent one seems to miss that particular build

**Comment by ymartin59** [comment]
@cytec That is right I forgot to publish `ffmpeg` for avoton with DSM 5 toolchain - I was not aware of `x64` shortcut to build them all once in a single package. See #3026 - by the way, if @dajoni expects to run such package on DSM 6, it has to be built with DSM 6 toolchain. So question to you @dajoni what is your DSM version running on DS415+/avaton ?

**Comment by dajoni** [comment]
Hi again.

Both my 216+ and 415+ is DSM 6.1.4-15217 Update 2 (latest).
